prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -122,6 +124,44 @@ class HttpUtilities {
                 .build()
         }
 
+        fun httpResponseHTML(
+            request: HttpRequestMessage<String?>,
+            responseBody: String,
+            httpStatus: HttpStatus,
+        ): HttpResponseMessage {
+            return request
+                .createResponseBuilder(httpStatus)
+                .body(responseBody)
+                .header(HttpHeaders.CONTENT_TYPE, htmlTextUTF8)
+                .build()
+        }
+
+        fun httpResponseImage(
+            request: HttpRequestMessage<String?>,
+            responseBody: ByteArray,
+            httpStatus: HttpStatus,
+        ): HttpResponseMessage {
+            return request
+                .createResponseBuilder(httpStatus)
+                .body(responseBody)
+                .header(HttpHeaders.CONTENT_TYPE, imagePNG)
```

Comment:
Maybe you could use the constants here? `io.ktor.http.ContentType.Image#getJPEG`

Reply:
yea, it seems both guava and ktor are among the dependencies

Reply:
OH, I see, can use the ktor resource (io.ktor.http.ContentType.Image#getJPEG etc.)